apps/web/src/pages/TalentNominations/NominationGroup/components/NominatedForList.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable formatjs/enforce-plural-rules */
+// NOTE: Not really plural, I guess?
```

Comment:
I agree, this is not a good use of the pluralization tool.  I recommend refactoring this - probably to conditionally concatenate the parens to the end of the message.  I feel like we often change the color, weight, or size of the font for these anyway.

Reply:
Yeah, I created [this ticket ](https://github.com/GCTC-NTGC/gc-digital-talent/issues/13583)but I'm not sure if I should roll this into it or how to deal with it :thinking:

Reply:
Nice, that looks like a good ticket.  :+1: